examples/auctions/blind_auction.vy

```diff
@@ -0,0 +1,63 @@
+# Blind Auction
+# Adapted to Vyper from [Solidity by Example](https://github.com/ethereum/solidity/blob/develop/docs/solidity-by-example.rst#blind-auction-1)
+
+# Custom struct for bids
+bid: {
+    blindedBid: bytes32,
+    deposit: wei_value
+}
+
+# Auction parameters
+beneficiary: public(address)
+biddingEnd: public(timestamp)
+revealEnd: public(timestamp)
+
+# Set to true at the end of auction, disallowing any new bids
+ended: public(bool)
+
+# State of bids
+bids: public([address])
```

Comment:
It looks like you are missing a "to" value for the mapping (we really need to improve this syntax), which I am guessing you mean to write `bids: public(bid[address])`

As noted above `bid` is not a named struct, so this will not work. The following will work:
```python
bids: public({
    blindedBid: bytes32,
    deposit: wei_value
}[address])
```